builds/linux/rhel/9/linux-rhel.pkrvars.hcl.example

```diff
@@ -21,3 +21,20 @@ vm_firmware              = "efi-secure"
 iso_datastore_path       = "iso/linux/rhel"
 iso_content_library_item = "rhel-9.4-x86_64-dvd"
 iso_file                 = "rhel-9.4-x86_64-dvd.iso"
+
+rpm_gpg_keys = [
+  "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+]
+
+yum_repositories = [
+  {
+    "name": "baseos",
+    "url": "http://url/path/to/repo",
+    "install": true
+  },
+  {
+    "name": "appstream",
+    "url": "http://url/path/to/repo",
+    "install": true
+  }
+]
```

Comment:
I think that this should be defined at a variable and only used if additional repos are enabled enabled.

Reply:
commented :)